src/Addon/Plugin/PluginCriteria.php

```diff
@@ -247,7 +264,7 @@ public function setCachePrefix($cachePrefix)
      */
     public function getCacheTtl()
     {
-        return array_get($this->collection, 'ttl', 60 * 24 * 7);
```

Comment:
`this->collection` is a `string` not an `array`. It simply holde the name of the CollectionClass. So this can't return anything!